src/app/components/icons/IconDrawer.tsx

```diff
@@ -69,6 +80,8 @@ export const IconDrawer: React.FC = () => {
     const history = useHistory();
     const dispatch = useDispatch();
     const classes = useStyles(theme);
+    const [iconSize, setIconSize] = React.useState(24);
```

Comment:
We will need to be careful here if somebody is switching between Material Icons and PX Blue icons with a size or color selected that is not supported by the new icon. We may need to reset this selection each time the icon is changed (if the icon class is different).